core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +59,15 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
+    mUserFileWriteCapacityReserved = Configuration
+        .getBytes(PropertyKey.USER_FILE_WRITE_CAPACITY_RESERVED_SIZE_BYTES);
+    return mCapacityBytes - mUsedBytes - mUserFileWriteCapacityReserved;
```

Comment:
The space reservation logic is specific to `LocalFirstAvoidEvictionPolicy`, so we should put the logic there instead of modifying `BlockWorkerInfo`